### PR TITLE
Changes in Renew Token

### PR DESCRIPTION
**Pull request by BindyaBhaskar**

### CODE REVIEW

File: spec/definitions/TokenRenewV0.yaml [hunk 1]
```diff
@@ -1,28 +1,55 @@
...
     title: Enterprise id
     description: Id of the enterprise resource
     type: string
     format: uuid
+    readOnly: true
   token:
     title: API Token
     description: API token for accessing esper APIs
     type: string
+    readOnly: true
```
Comment (abdulMuneerM) [inline spec/definitions/TokenRenewV0.yaml:24]: why is this field read only?

**Review by tr0n-p46 — APPROVED** [review]

File: spec/definitions/TokenRenewV0.yaml (hunk 1 above)
Comment (BindyaBhaskar) [reply spec/definitions/TokenRenewV0.yaml:24]: Will change.

Comment (tarun91vas) [reply spec/definitions/TokenRenewV0.yaml]: @abdulMuneerM  I think read only should be fine. readonly means api user will not be able to update any of these parameters.

Comment (abdulMuneerM) [reply spec/definitions/TokenRenewV0.yaml]: Ok

**Review by abdulMuneerM — APPROVED** [review]

**Pull request merged**